.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "airbnb",
+  "rules": {
```

Comment:
Since you are going to be using `ReactJS` in this codebase, extending your style config by `airbnb` is not enough, for `airbnb` to work properly for `ReactJS` codes, you need to add these pluging to your eslint configuration:
```
eslint-plugin-import
eslint-plugin-react
eslint-plugin-jsx-a11y
```
You can check the documentation here for more info: https://www.npmjs.com/package/eslint-config-airbnb